index.html

```diff
@@ -183,16 +193,17 @@
       </tbody>
     </table>
 
-    <p class="note">The <code>gAMA</code> and embedded ICC profile are provided solely for compatibility with processors that do
-    not conform to this specification.</p>
+    <p class="note">The <code>gAMA</code> chunk, <code>cHRM</code> chunk, and embedded ICC profile specified in the table above are
+    not necessarily accurate, and, even if accurate, their processing is not required to reproduce the image as intended by the
+    author. They are provided for graceful fallback for implementations that do not recognize the semantics associated with a
+    <code>iCCP</code> Chunk Profile Names defined by this specification. Future versions of this specification might include
+    additional ICC profiles, e.g. to reflect improvements in ICC profile capabilities.</p>
```

Comment:
Like that part, it allows a V5 profile to be added later which actually describes the image.